backend/EasySpeak.Core/EasySpeak.Core.WebAPI/firebaseconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "easy-meets-id-01",
+  "private_key_id": "e62cc972ce15e6979d580a72eac10887b8d5160e",
```

Comment:
This is not good practice to push such files to the repository

Reply:
Only two variables remained